### DISCUSSION TITLE
Add performance metrics

### DISCUSSION THREAD

**Discussion by Nick-Mazuk** [supabase/supabase#4706] [Feature Requests]
# Feature request

## Is your feature request related to a problem? Please describe.

Wow, the new [metrics in the dashboard](https://supabase.io/blog/2021/07/29/supabase-reports-and-metrics) are unexpected and awesome!

It would be great to also add performance metrics to these list of metrics. It could be instrumental in finding slow queries to rewrite to speed them up. Or even finding out which columns should have indexes.

## Describe alternatives you've considered

It is possible to either manually setup a plugin to do this, but baking it into Supabase and combining it with the new reports would be better.

Additionally, you could also just monitor network requests from the website itself (similar to how Firebase analytics works), but that kind of monitoring is only good for identifying performance problems, not identifying the root causes of those problems.

## Additional context

Great launch week! Can't wait to see what tomorrow's surprise is.

**Comment by kiwicopple** [comment]
We'll definitely expose some `pg_stat_statements` details (and a few other nice things)
